integration_tests/pages/page.ts

```diff
@@ -1,19 +1,51 @@
+import 'cypress-axe'
+import { Result } from 'axe-core'
+
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
   static verifyOnPage<T>(constructor: new () => T): T {
     return new constructor()
   }
 
-  constructor(private readonly title: string) {
+  constructor(
+    private readonly title: string,
+    private readonly skipA11y = false,
```

Comment:
not a great fan of booleans in constructors - difficult to know what the `super('Sign in', true)` means without having a look at the implementation

Reply:
Would be easier as an object then at least you can have meaningful attribute names in there and then use a boolean.